jmix-flowui/flowui/src/main/java/io/jmix/flowui/model/impl/CollectionLoaderImpl.java

```diff
@@ -101,6 +126,9 @@ protected boolean _load() {
             }
         }
 
+        DataLoaderMonitoringInfo info = monitoringInfoProvider.apply(this);
```

Comment:
it may appears that `monitoringInfoProvider` is not set. it's better to check state and throw IllegalStateException if not set, e.g. List actions do the same to check if required target is set.

Reply:
Now it is always set.